### PR TITLE
Updated initClient for correct connection status

### PR DESCRIPTION
**Pull request by sp-suresh**

### CODE REVIEW

File: examples/voter/voter/models/volt.js [hunk 1]
```diff
@@ -179,15 +179,19 @@ exports.initClient = function(startLoop) {
...
+        if(startLoop == true) {
+          setInterval(logResults, statsLoggingInterval);
+          voteInsertLoop();
+        } else {
+          voltInit();
+        }
+      }
+      else{
+        util.log(`Unexpected status while initClient: ${VoltConstants.STATUS_CODE_STRINGS[code]}`);
+        process.exit();
```
Comment (vittvolt) [inline examples/voter/voter/models/volt.js]: exit with code 1 ?

**Comment by sp-suresh** [comment]
Are there any changes need to be done?, Why so much delay in the process?

File: examples/voter/voter/models/volt.js [hunk 2]
```diff
@@ -179,15 +179,19 @@ exports.initClient = function(startLoop) {
...
-      util.log('Node connected to VoltDB');
-      
-      if(startLoop == true) {
-        setInterval(logResults, statsLoggingInterval);
-        voteInsertLoop();
-      } else {
-        voltInit();
+      if(code == VoltConstants.STATUS_CODES.SUCCESS){
+        util.log('Node connected to VoltDB');
+      if(startLoop) {
```
Comment (nshi) [inline examples/voter/voter/models/volt.js]: Thank you for the patch, @sp-suresh. The logic looks good. If you don't mind fixing the indentation here, I will merge your PR.

**Comment by sp-suresh** [comment]
Are there any changes need to be done?

**Pull request merged**